src/components/work/overview/bookmarkDropdown/BookmarkDropdown.js

```diff
@@ -18,13 +30,28 @@ export function BookMarkMaterialSelector({
   size = { w: 7, h: 7 },
   className,
   title,
+  singleManifestation,
   editions,
 }) {
-  const { bookmarks, setBookmark, isLoading } = useBookmarks();
-  const [active, setActive] = useState(false);
-  const [options, setOptions] = useState(
-    materialTypes.map((mat) => formatMaterialTypesToCode(mat))
-  );
+  const manifestations = editions;
```

Comment:
skal vi ikke bare kalde editions for manifestations i parameterne?

Reply:
singleManifestation behøver vi faktisk ikke give med som parameter, det er jo manifestations/editions.length?